### PR TITLE
SSB: Nerfs/Buffs

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: mods/seasonal/statuses.js [hunk 1]
```diff
@@ -465,7 +465,7 @@ let BattleStatuses = {
 			this.add(`-anim`, target, 'Geomancy', target);
 			target.formeChange(forme);
 			target.setAbility('Anticipation');
```
Comment (ghost) [inline mods/seasonal/statuses.js]: *cough*

Comment (KrisXV) [reply mods/seasonal/statuses.js]: How did I miss this

Comment (HoeenCoder) [reply mods/seasonal/statuses.js]: dont forget the hint too

Comment (ghost) [reply mods/seasonal/statuses.js]: The hint is fine. why this isnt adaptability is beyond me

**Comment by KrisXV** [comment]
I tried my best to get the prismatic terrain message to stop double-posting, but noting really worked and it doesnt seem to cause any issues other than it posts twice so thats better than nothing for now

File: mods/seasonal/moves.js [hunk 2]
```diff
@@ -3798,6 +3868,16 @@ let BattleMovedex = {
 			this.add('-anim', source, "Air Slash", target);
 			this.add('-anim', source, "Air Slash", target);
 		},
+		secondaries: [
+			{
+				chance: 20,
+				volatileStatus: 'flinch',
+			},
```
Comment (ghost) [inline mods/seasonal/moves.js:3880]: This move traps the target, btw

**Pull request merged**